src/Freya.Core/Integration.fs

```diff
@@ -18,12 +18,12 @@
 //
 //----------------------------------------------------------------------------
 
-[<AutoOpen>]
-module Freya.Core.Integration
+module Freya.Integration
```

Comment:
Do these need to live under separate spaces? Anything in Freya.Core should be Freya.Core.* I think, generally auto-opened as well. I'd be quite happy to lose the Freya.Pipeline and Freya.Integration namespaces and subsume them in to Freya.Core I think?